.ci_support/linux_64_.yaml

```diff
@@ -45,14 +43,16 @@ libaec:
 libarchive:
 - '3.7'
 libarrow:
-- '13'
 - '14'
 - '15'
 - '16.1'
+- '17.0'
 libcurl:
 - '8'
 libdeflate:
 - '1.20'
+libgdal:
+- '3.9'
```

Comment:
This looks suspicious to me, @isuruf have any thoughts?

Reply:
It's an old bug in conda-build, but harmless. It's okay to ignore